common/source/docs/common-frsky-passthrough.rst

```diff
@@ -0,0 +1,62 @@
+.. _common-frsky-passthrough:
+
+====================================
+Passthrough FrSky Telemetry Protocol
+====================================
+
+Overview
+========
+
+Passthrough FrSky Telemetry is the latest protcol available in ArduPilot and it allows the transmission of raw data directly to OpenTX without any preprocessing by OpenTX and without the need for discovery. The passthrough telemetry protocol is optimized specifically for the FrSky datalink bandwidth and enhancements include having multiple data elements in a single telemetry packet and using floating point representation (e.g. we don't really need centimeter precision altitude to be displayed on screen when flying at 100m). This results in an improved data refresh rate such that information such as attitude (roll, pitch, yaw) is displayed without any perceptible lag.
+
+Compared to regular FrSky telemetry, passthrough telemetry contains ArduPilot specific information such as flight modes, error messages, warnings, and failsafes.
+
+ArduPilot's Passthrough FrSky telemetry protocol is an open protocol that is free to use. `FlightDeck <http://www.craftandtheoryllc.com/product/flightdeck-taranis-app-for-ArduPilot-apmcopter/>`__ is an OpenTX app that uses passthrough telemetry.
+
+The ArduPilot passthrough telemetry specification including data packets and message rates is available as a spreadsheet on `Google Sheets <https://docs.google.com/spreadsheets/d/1HIfioe-EGTOtH5rOdexGlAz2E5HnCvZLm0EG6rfjNSI/edit?usp=sharing>`__.
```

Comment:
@lvale, I had that originally in a table on the page but the table is really big and doesn't fit because of the font size. I'm open to suggestions on this. To interpret the data you can use this table. Since this is passthrough telemetry, you have to write the code to interpret the data in your script/app. It's the same computation as in the ArduPilot side but in reverse. Once you have the ID, you know what computation to do, using the table.